pkg/scheduler/helper.go

```diff
@@ -0,0 +1,110 @@
+package scheduler
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"k8s.io/klog/v2"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/pkg/util"
+)
+
+func weatherResourceBindingNeedSchedule(rb *workv1alpha2.ResourceBinding) (bool, string) {
```

Comment:
Does the `doScheduleClusterBinding` also need to be modified accordingly?

Reply:
updated